src/app/investment-wallet/components/buy-and-sell/buy-and-sell.component.ts

```diff
@@ -0,0 +1,16 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'buy-and-sell',
```

Comment:
Nome do componente ta errado, tem que começar com o nome do módulo
investment-module-buy-and-sell